diagrammer/python/scene.py

```diff
@@ -463,6 +464,66 @@ def gps(self) -> None:
         else:
             self._width = self._height = 0
 
+        # make overlapping references' path BEZIER_CLOCK
+        ref_by_angle = defaultdict(list)
+
+        for ref in self._references:
+            # get canonical degree, which is floored and [0, 180)
+            canonical_degree = math.degrees(ref.get_head_angle())
+
+            while not (0 <= canonical_degree < 180):
```

Comment:
isn't this basically turning the head angle into the tail angle? right cause it's basically just shifting the "origin" of the angle measurement from head to tail and geometrically that's basically just transforming head angle into tail angle

Reply:
also I think it'd be better design to make this a part of the `Arrow` class so that you can reuse it outside of gps if needed (and so that gps doesn't calculate a property of the arrow)

Reply:
I feel like this is a gps specific thing though not an Arrow thing because only this specific part of GPS needs "either the head or tail, whichever one's between 0 and 180"